packages/synapse-react-client/src/components/FullTextSearch.tsx

```diff
@@ -60,50 +60,58 @@ export const FullTextSearch: React.FunctionComponent<FullTextSearchProps> = ({
     }
   }
 
-  const handleChange = (event: React.FormEvent<HTMLInputElement>) => {
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
     searchInputRef.current?.setCustomValidity('')
     setSearchText(event.currentTarget.value)
   }
 
   return (
-    <div className={`QueryWrapperSearchInput`}>
+    <div className={`QueryWrapperFullTextSearchInput`}>
       <Collapse in={showSearchBar} timeout={{ enter: 300, exit: 300 }}>
-        <div className="QueryWrapperSearchInput__helppopoverwrapper">
-          <form
-            className="QueryWrapperSearchInput__searchbar"
-            onSubmit={search}
-          >
-            <span className="QueryWrapperSearchInput__searchbar__searchicon">
-              <IconSvg icon="search" />
-            </span>
-            <input
-              ref={searchInputRef}
-              minLength={MIN_SEARCH_QUERY_LENGTH}
-              onChange={handleChange}
-              placeholder="Enter Search Terms"
-              value={searchText}
-              type="text"
-            />
-            {searchText.length > 0 && (
-              <button
-                className="QueryWrapperSearchInput__searchbar__clearbutton"
-                type="button"
-                onClick={() => {
-                  setSearchText('')
-                }}
-              >
-                <IconSvg icon="close" />
-              </button>
-            )}
-          </form>
-          <div className="QueryWrapperSearchInput__helppopover">
-            <HelpPopover
-              markdownText={helpMessage}
-              helpUrl={helpUrl}
-              placement="left"
-            />
-          </div>
-        </div>
+        <form onSubmit={search}>
```

Comment:
For both of these components, use MUI TextField and put the popover in the `endAdornment`. This lets the input field span the full width of the column, and matches the nice clean edge line that we're trying to establish in PORTALS-2636

Reply:
"`endAdornment`"?  Ok